Source/Master/lib/JsonBuilder/JsonBuilder.cpp

```diff
@@ -0,0 +1,66 @@
+#include "JsonBuilder.hpp"
+#include "json.hpp"
+using json = nlohmann::json;
+
+JsonBuilder::JsonBuilder()
```

Comment:
Constructor without contents? Use `= default` in header?
"A defaulted default constructor is specifically defined as being the same as a user-defined default constructor with no initialization list and an empty compound statement."
https://stackoverflow.com/questions/20828907/the-new-syntax-default-in-c11

Reply:
Fixed